src/main/java/seedu/address/model/person/Schedule.java

```diff
@@ -0,0 +1,113 @@
+package seedu.address.model.person;
+
+import seedu.address.model.person.exceptions.InvalidTimeException;
+import seedu.address.model.person.exceptions.OverlapTaskException;
+
+/**
+ * Represents the schedule for the staff, which contains all the task for the staff.
+ * Built as a interval tree.
+ */
+public class Schedule {
+    private Task taskNode = null;
```

Comment:
Sorry, please clarify how Task and Schedule work together? Why not have an array of Tasks where each Task in the array represents one shift? This way, can't you only express one Task/shift per schedule (sorry if I am interpreting it wrong)

Reply:
At the beginning, I was thinking to build the schedule like a schedule is a tree and task is its node. So in that case it may be more efficient to compare which one is earlier or whether there are overlapped tasks. However, since we have changed to a simpler version, I will change it soon.